test-app/openapi/v1/openapi.json

```diff
@@ -204,14 +234,19 @@
           "content": {
             "application/json": {
               "schema": {
-                "oneOf": [
-                  {
-                    "$ref": "#/components/schemas/blog"
-                  },
-                  {
-                    "$ref": "#/components/schemas/flexible_blog"
+                "type": "object",
+                "properties": {
+                  "flexible_blog": {
```

Comment:
@stevegeek I believe the `type` nor `properties` should not be there.

It renders the request example including the `flexible_blog`, which is changes the content from what is expected.

Reply:
ok will take a look. Probably Im mixing up between form data fields and the body as a whole which is what is expected here